src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 10px;
+}
+
+.bar {
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  display: inline-block;
+  width: 100%;
+  position: relative;
+}
+
+.bar__input {
+  font-family: Avenir, Times, sans-serif;
+  font-weight: 300;
+}
+
+.bar__img__big {
+  position: absolute;
+  top: 24px;
+  left: 24px;
+  height: 19px;
+}
+
+.bar__input__big {
+  font-size: 16px;
+  padding-left: 66px;
+  border: 0;
+  height: 68px;
+}
```

Comment:
checklist point 2
these styles to the bar__input, delete styles for .bar__input__big, leave styles for .bar__input__small
the same for similar cases